std_msgs/Header header

float64 left_radius
float64 right_radius
float64 centre_offset